includes/Story_Post_Type.php

```diff
@@ -222,21 +222,19 @@ public function show_admin_bar( $show ) {
 	}
 
 	/**
-	 * Highjack editor with custom editor.
+	 * Replace default post editor with our own implementation.
+	 *
+	 * @codeCoverageIgnore
```

Comment:
Why ignore?

Reply:
It's explained in PR description.

Can't really test a method, so no point in counting it towards coverage.